pkgs/by-name/pa/passt/package.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, lib, fetchgit }:
+
+stdenv.mkDerivation {
+  pname = "passt";
+  version = "0.2023_11_10.5ec3634";
+  src = fetchgit {
+    url = "git://passt.top/passt";
+    rev = "5ec3634b07215337c2e69d88f9b1d74711897d7d";
+    hash = "sha256-76CD9PYD/NcBkmRYFSZaYl381QJjuWo0VsNdh31d6/M=";
+  };
+  nativeBuildInputs = [ ];
+  buildInputs = [];
```

Comment:
```suggestion
```

Reply:
This was in https://github.com/NixOS/nixpkgs/pull/265409 which is now in master, sorry.